src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -153,22 +162,30 @@ def acr_show(registry_name, resource_group_name=None):
     return client.get(resource_group_name, registry_name)
 
 
-def acr_update_get(client):  # pylint: disable=unused-argument
+def acr_update_get(client,  # pylint: disable=unused-argument
+                   registry_name,
+                   resource_group_name=None):
     """Returns an empty RegistryUpdateParameters object.
+    :param str registry_name: The name of container registry
+    :param str resource_group_name: The name of resource group
     """
-    return RegistryUpdateParameters()
+    try:
+        registry_sku_validation(registry_name, resource_group_name)
+        return ManagedRegistryUpdateParameters()
+    except:  # pylint: disable=bare-except
```

Comment:
```
except Exception:
```

reference: https://stackoverflow.com/questions/18982610/difference-between-except-and-except-exception-as-e-in-python